src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
Updated the apidoc portion of the file. We also should consider writing additional docs outlining the different guarantees that apply to notifications from the ledger and explain how a client could implement robust notifications on top of the ledger if needed by an application. (I would be in favor of open sourcing a client implementing robust notifications actually).